cloudevents/tests/test_event_pipeline.py

```diff
@@ -77,7 +77,7 @@ def test_object_event_v1():
     _, structured_body = m.ToRequest(event)
     assert isinstance(structured_body, bytes)
     structured_obj = json.loads(structured_body)
-    error_msg = f"Body was {structured_body}, obj is {structured_obj}"
+    error_msg = f"Body was {structured_body!r}, obj is {structured_obj}"
```

Comment:
Lint error warning that bytes would be formatted as a quoted repr, unlike str, so it demands explicit repr (`!r`) to avoid ambiguity.